From 6d9bb088764e5635838653d2645d9be9e51da682 Mon Sep 17 00:00:00 2001
From: Maxim Uvarov <maxim.uvarov@linaro.org>
Date: Thu, 25 Feb 2021 10:22:20 +0000
Subject: [PATCH] pseudo: Bump to rev 60e25a3

Use patch from mailing list to update pseudo to support
gcc10 build.
https://lists.openembedded.org/g/openembedded-core/message/148498

Signed-off-by: Maxim Uvarov <maxim.uvarov@linaro.org>
---
 meta/recipes-devtools/pseudo/pseudo_git.bb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meta/recipes-devtools/pseudo/pseudo_git.bb b/meta/recipes-devtools/pseudo/pseudo_git.bb
index 0ba7b50355..17bd02c27c 100644
--- a/openembedded-core/meta/recipes-devtools/pseudo/pseudo_git.bb
+++ b/openembedded-core/meta/recipes-devtools/pseudo/pseudo_git.bb
@@ -6,7 +6,7 @@ SRC_URI = "git://git.yoctoproject.org/pseudo;branch=oe-core \
            file://fallback-group \
            "
 
-SRCREV = "8317c0ab172db47dabcef909bae02cd77b1f1010"
+SRCREV = "60e25a36558f1f07dcce1a044fe976b475bec42b"
 S = "${WORKDIR}/git"
 PV = "1.9.0+git${SRCPV}"
 
-- 
2.17.1

